_posts/fr/newsletters/2024-02-28-newsletter.md

```diff
@@ -0,0 +1,275 @@
+    temporel relatif offre une période de contestation (voir ci-dessous) ; la vérification du montant
+    exact garantit qu'aucun fonds ne soit retiré sans autorisation ; et la vérification de l'adresse
+    garantit qu'un voleur ne peut pas changer une adresse de retrait légitime en sa propre adresse à la
+    dernière minute (un problème avec tous les coffres pré-signés dont nous avons connaissance, voir
+    le [Bulletin #59][news59 vaults]).
+
+  - *Dépense illégitime :* Mallory déclenche une dépense en créant une transaction initiatrice comme
+    décrit ci-dessus. La [tour de guet][topic watchtowers] d'Alice réalise que la dépense est illégitime
+    pendant la période de contestation (par exemple, le délai de 20 blocs) et crée une transaction de
+    re-coffrage avec deux entrées et une sortie ; les entrées sont la première sortie de la transaction
+    déclencheuse et une entrée payant des frais ; la sortie est un retour au coffre. Étant donné que la
+    transaction de re-coffrage n'a qu'une seule sortie mais que les conditions de retrait du script
+    exigent une dépense à partir d'une transaction initiatrice avec deux sorties, Mallory est incapable
+    de voler les fonds d'Alice.
+
+    Puisque l'argent est retourné au même script de coffre d'où il est parti, Mallory peut toujours
+    créer une autre transaction initiatrice et forcer Alice à passer par le même cycle encore et
+    encore, entraînant des coûts de frais pour Mallory et Alice. La [documentation étendue][cat vault
+    readme] de Rijndael pour le projet note que vous voudriez probablement permettre à Alice de dépenser
+    l'argent vers un script différent dans ce cas, et que les idées derrière sa construction le
+    permettent mais cela n'est pas actuellement implémenté pour simplifier.
+
+  Ces coffres basés sur CAT peuvent être comparés aux types de coffres pré-signés que nous pouvons
+  créer aujourd'hui sans changements de consensus et les coffres `OP_VAULT` de style [BIP345][] qui
+  fourniraient l'ensemble de fonctionnalités de coffre le mieux connu si leur support était ajouté
+  dans un soft fork.
+
+  <table>
+  <tr>
+    <th></th>
+    <th>Pré-signé</th>
+    <th markdown="span">BIP345 `OP_VAULT`</th>
+    <th markdown="span">`OP_CAT` avec schnorr</th>
+  </tr>
+
+  <tr>
+    <th>Disponibilité</th>
+    <td markdown="span">**Maintenant**</td>
+  <td markdown="span">Nécessite un soft fork de `OP_VAULT` et [OP_CTV][topic op_checktemplateverify]</td>
+    <td markdown="span">Nécessite un soft fork de `OP_CAT`</td>
+  </tr>
+
+  <tr>
+    <th markdown="span">Attaque de remplacement d'adresse de dernière minute</th>
+    <td markdown="span">Vulnérable</td>
+    <td markdown="span">**Non vulnérable**</td>
+    <td markdown="span">**Non vulnérable**</td>
+  </tr>
+  <th markdown="span">Retraits partiels de montant</th> <td markdown="span">Seulement si préarrangé</td>
```

Comment:
@Copinmalin please revisit the table formatting here, tr and th tags

Reply:
OK, I hope is good this time